src/main/java/com/company/mvcfreemarker/controller/UserController.java

```diff
@@ -0,0 +1,21 @@
+package com.company.mvcfreemarker.controller;
+
+import com.company.mvcfreemarker.entity.User;
+import io.jmix.core.UnconstrainedDataManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@RequestMapping("/portal/users")
+public class UserController {
+    @Autowired
+    private UnconstrainedDataManager dataManager;
+
+    @RequestMapping(method = {RequestMethod.GET, RequestMethod.POST})
+    public String getUsers(@ModelAttribute("model") ModelMap model) {
+        model.addAttribute("users", dataManager.load(User.class).all().list());
```

Comment:
Let's discuss it, but probably it is worth introducing the UserDTO class which contains only attributes displayed on the page. In this case, you'll query a list of User entities, convert them to the list of UserDTO and then put a list of UserDTO to the model.